<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="widget">
          <attribute name="title">
           <string>Entidades</string>
          </attribute>
          <widget class="QComboBox" name="comboBox">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>30</y>
             <width>101</width>
             <height>22</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>Usuario</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sucursal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Departamento</string>
            </property>
           </item>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>49</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Entidad:</string>
           </property>
          </widget>
          <widget class="QGroupBox" name="groupBoxDetalleEntidad">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>621</width>
             <height>221</height>
            </rect>
           </property>
           <property name="title">
            <string>Detalle Entidad</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frameCambio">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QLabel" name="label_2">
               <property name="geometry">
                <rect>
                 <x>220</x>
                 <y>90</y>
                 <width>181</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="text">
                <string>Frame de cambio</string>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QPushButton" name="BtnSeleccionar">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>30</y>
             <width>75</width>
             <height>24</height>
            </rect>
           </property>
           <property name="text">
            <string>Seleccionar</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="Tab1">
          <attribute name="title">
           <string>Transacciones</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Reportes/Consultas</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Utilidades</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Ayudas</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
